### PR TITLE
✨ feat(components,i18n,content): fix, refactor, update mdx content, i…

### PR DESCRIPTION
**Pull request by anIcedAntFA**
…18n, messages, velite

### CODE REVIEW

File: content/en/articles/hello-world.mdx [hunk 1]
```diff
@@ -5,6 +5,7 @@ slug: hello-world
 date: 1992-02-25 13:22
 cover: /images/posts/cover.jpg
 video: /videos/posts/video.mp4
+locale: en
 authors:
   - ngockhoi96
 ---
```
Comment (anIcedAntFA) [inline content/en/articles/hello-world.mdx:11]: **suggestion**: remove unused hello-world mdx file

File: src/app/[locale]/articles/[slug]/page.tsx [hunk 2]
```diff
@@ -28,22 +29,38 @@ interface IPostProps {
 //   return doc;
 // }
```
Comment (anIcedAntFA) [inline src/app/[locale]/articles/[slug]/page.tsx:30]: **suggestion**: remove unused comment code

File: src/app/[locale]/articles/[slug]/page.tsx [hunk 3]
```diff
@@ -28,22 +29,38 @@ interface IPostProps {
...
 
-function PostsPage({ params }: IPostProps) {
-  const post = getPostBySlug(params.slug);
+function ArticlesPage({ params: { locale, slug } }: ArticlesPageProps) {
+  const article = getArticleBySlug(slug, locale);
 
   const t = useTranslations('articles.components.tableOfContents');
 
-  if (post == null) {
+  if (article == null) {
```
Comment (anIcedAntFA) [inline src/app/[locale]/articles/[slug]/page.tsx:63]: **suggestion**: simplify by checking with `!article`

File: src/components/common/link/link.tsx [hunk 4]
```diff
@@ -18,7 +17,7 @@ function Link<TPathname extends Pathname>(
   }
 
   return (
-    <LocalizedLink href={href} {...restProps}>
+    <LocalizedLink href={''} {...restProps}>
```
Comment (anIcedAntFA) [inline src/components/common/link/link.tsx:20]: **suggestion**: add `href`

**Review by ngockhoi96-dev — APPROVED** [review]

**Pull request merged**